testing/e2e/tools/download.go

```diff
@@ -0,0 +1,55 @@
+//go:build linux || darwin
+// +build linux darwin
+
+package tools
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func DownloadElasticAgent(version string) error {
+	dirToInstall, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	fileName, destFileName, err := tarName(version)
+	if err != nil {
+		return err
+	}
+	agentTarPath := fmt.Sprintf("%s/%s", dirToInstall, destFileName)
+	err = DownloadFile(agentTarPath, fmt.Sprintf("https://artifacts.elastic.co/downloads/beats/elastic-agent/%s", fileName))
+
+	if err != nil {
+		return err
+	}
+	_, err = exec.Command("tar", "-xvf", agentTarPath).Output()
+
+	return err
+}
+
+func DownloadFile(filepath string, url string) error {
+	log.Info("Downloading Elastic Agent...")
+	log.Info(url)
```

Comment:
[Suggestion]
merge it on a single log. It might not be obvious to everyone that the url printed after is the one used to download the file.
```suggestion
	log.Infof("Downloading Elastic Agent from %s", url)
```